client/src/components/ProjectSettings.vue

```diff
@@ -53,12 +55,15 @@ export default defineComponent({
           await djangoRest.setGlobalSettings({
             import_path: importPath.value.trim(),
             export_path: exportPath.value.trim(),
+            anatomy_orientation: undefined,
```

Comment:
Since the server-side logic is expecting this key not to be set, I think the more straightforward expression of that is to just omit it here.
```suggestion
```

The "right" thing to do for this request would be to support a partial update of this object via an HTTP PATCH, but that's out of scope here, and not really that important in this context.

Reply:
Changed in [463db2b](https://github.com/OpenImaging/miqa/pull/497/commits/463db2b8c82c4e83c5e6dc928baec4287c22e3b6)